gc/base/MarkingScheme.hpp

```diff
@@ -139,13 +141,7 @@ class MM_MarkingScheme : public MM_BaseVirtual
 	MMINLINE bool
 	inlineMarkObject(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, bool leafType=false)
 	{
-		bool didMark = false;
```

Comment:
Because the code was hard to read. Unnecessarily verbose, spurious use of local variables obfuscates how result is obtained. Current form is much more clear and explicit IMO.